database/migration/src/main/resources/changesets/restart_and_recover_migration.xml

```diff
@@ -76,6 +76,23 @@
         </sql>
     </changeSet>
 
+    <changeSet id="JOB_STORE_RESULT_SIMPLETON_VALUE_NULLABLE" author="mcovarr">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="JOB_STORE_RESULT_SIMPLETON"/>
+        </preConditions>
+        <comment>
+            This changeset is a retroactive insertion into the changeset order to enable migration of
+            null output values in a 0.19 Cromwell installation to the job store simpleton table introduced
+            in pluggable backend Cromwell.  Because this is a retroactive insertion, the "tableExists"
+            precondition keeps this changeset from running on a database that has already been migrated to
+            the pluggable backend schema (this migration is unnecessary in a database that has already been
+            migrated to the pluggable backend schema since a later changeset also drops this constraint).
```

Comment:
The later migration dropping the constraint doesn't fail if it has already been dropped by this one ?